tests/flow/test_serializations.py

```diff
@@ -161,3 +161,68 @@ def test_v1_tensor(self):
         self.env.assertEqual([tensor_type, tensor_shape], [b"INT32", [2, 1]])
         values = con.execute_command('AI.TENSORGET', key_name, 'VALUES')
         self.env.assertEqual(values, [1, 2])
+
+
+class test_v2_rdb_load:
```

Comment:
for future-proofing, please have a v2 rdb dumped and set as strings in this file.